### PR TITLE
Add ENULS Chain (mainnet & testnet) 

### PR DESCRIPTION
**Pull request by niels1286**
[I make a pin the icon at : https://ipfs.io/ipfs/QmYz8LK5WkUN8UwqKfWUjnyLuYqQZWihT7J766YXft4TSy?filename=ENULS.svg](https://github.com/ethereum-lists/chains/pull/1588)

### CODE REVIEW

File: _data/chains/eip155-119.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "ENULS Mainnet",
+  "chain": "ENULS",
+  "network": "mainnet",
```
Comment (ligi) [inline _data/chains/eip155-119.json:4]:
```suggestion
```

File: _data/chains/eip155-120.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+  "name": "ENULS Testnet",
+  "chain": "ENULS",
+  "network": "testnet",
```
Comment (ligi) [inline _data/chains/eip155-120.json:4]:
```suggestion
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
short name must be unique - but found `ENULS` more than once

**Comment by niels1286** [comment]
> 

I have fixed this problem, please deal with it again, thank you

**Comment by niels1286** [comment]
I update the code format . All matched files use Prettier code style!

**Pull request merged**